_posts/2019-07-24-cockpit-199.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Cockpit 199
+author: kkoukiou
+date: 2019-07-24
+tags: cockpit linux
+slug: cockpit-199
+category: release
+summary: Cockpit with Logs and Services and HWInfo design updates
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 199.
+
+### Redesigned logs
+
+Restyled logs across all of Cockpit.
+
+![service-logs](/images/service-logs.png)
+
+### Services: Design and accesibility improvements
+
+The enablement and runtime status of units is now represented in a clearer way.
+
+![service-details](/images/service-details.png)
+
+In addition the tabs in Service page are now accessible.
+
+### System: Show DIMM information on Hardware Info page
+
+The Hardware Info page now can show information about DIMM memory, such as size, rank, speed etc.
```

Comment:
"DIMM memory" is redundant, as one of the Ms stands for "memory".

I'd suggest:

```suggestion
The Hardware Info page now displays memory-related information for DIMMs, such as: size, rank, speed, and other details.
```